source/bson-binary-vector/bson-binary-vector.md

```diff
@@ -184,6 +184,7 @@ Drivers MUST validate vector metadata and raise an error if any invariant is vio
 
 - Padding MUST be 0 for all dtypes where padding doesn’t apply, and MUST be within \[0, 7\] for PACKED_BIT.
 - A PACKED_BIT vector MUST NOT be empty if padding is in the range \[1, 7\].
+- For a PACKED_BIT vector, bits lower than the given padding (those ignored) must be zero.
```

Comment:
Is this for Encoding and Decoding?

Reply:
Yes. That's what we came to, and exceptions are thrown in the Python implementation in both cases.

Reply:
Right, it's specified a few lines after:
`Drivers MUST perform this validation when...`